source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -335,25 +368,18 @@ The SDAM specification defines `when
 <https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#connection-pool-creation>`_
 the driver should create connection pools.
 
-Once a pool is created, if minPoolSize is set, the pool MUST immediately begin
-being `populated
-<#populating-the-pool-with-a-connection-internal-implementation>`_ with enough
-`Connections <#connection>`_ such that totalConnections >= minPoolSize. This
-MUST be done in a non-blocking manner, such as via the use of a background
-thread or asynchronous I/O. See `Populating the Pool with a Connection
-<#populating-the-pool-with-a-connection-internal-implementation>`_ for more
-details on the steps involved.
+When a pool is created, its state MUST initially be set to "paused". Even if
+minPoolSize is set, the pool MUST NOT begin being `populated
+<#populating-the-pool-with-a-connection-internal-implementation>`_ with
+`Connections <#connection>`_ until it has been marked as "ready". Monitoring threads will
+ready the pool once they have successfully performed a check and
+transitioned the server to a known ServerType.
```

Comment:
`Monitoring threads` -> `SDAM will mark the pool as "ready" on each successful check.`

Also I think it would help to link to the SDAM `Connection Pool Management` section.

Reply:
done